proto/gen/einride/example/freight/v1/freight_service_aiptest.pb.go

```diff
@@ -84,7 +85,11 @@ func (fx *ShipperTestSuiteConfig) testCreate(t *testing.T) {
 		assert.NilError(t, err)
 		assert.Check(t, msg.CreateTime != nil)
 		assert.Check(t, !msg.CreateTime.AsTime().IsZero())
-		assert.Check(t, !msg.CreateTime.AsTime().After(time.Now()))
+		if runtime.GOOS == "darwin" {
+			assert.Check(t, !msg.CreateTime.AsTime().After(time.Now().Add(1*time.Second)))
+		} else {
+			assert.Check(t, time.Since(msg.CreateTime.AsTime()) < time.Second)
```

Comment:
I think checking if the Create method took less than 1s  is not the right assertion , since you can do stuff that takes more than one second, which is OK in some applications, or you machine is under much load and then it takes more than 1 second to complete.
I would instead rewrite this as:
```go
        beforeCreate := time.Now()
	msg, err := fx.service.CreateShipper(fx.ctx, &CreateShipperRequest{
			Shipper:   fx.Create(),
			ShipperId: userSetID,
		})
		assert.NilError(t, err)
		assert.Check(t, msg.CreateTime != nil)
		assert.Check(t, !msg.CreateTime.AsTime().IsZero())
		if runtime.GOOS == "darwin" {
			assert.Check(t, !msg.CreateTime.AsTime().After(time.Now().Add(1*time.Second)))
		} else {
			assert.Check(t, msg.CreateTime.AsTime().After(beforeCreate))
		}
```
What do you think about this?

Reply:
Nice catch @thall! Yeah the original code should definitely be generated here:

```go
		f.P(ident.AssertCheck, "(t, !msg.CreateTime.AsTime().After(", ident.TimeNow, "()))")
```

Reply:
Yeah, great catch - I knew this wasn't a great solution but of course we should try and fix this more reliably. 👍 Thanks for pushing on this @thall ⭐ 😄